lvsfunc/comparison.py

```diff
@@ -164,31 +164,31 @@ class Tile(Comparer):  # TODO: check this documentation will read correctly on t
     Think of this as a 2-dimensional array of `0` or `1` with `0` representing an empty slot and `1` representing the next clip in the sequence.
 
     If `arrangement` is not specified, the function will attempt to fill a square with dimensions `n x n`
-    where `n` is equivalent to ``math.ceil(math.sqrt(len(clips))`` .
-
-    For example, for 3 clips, the automatic arrangement becomes:
-    [
-     [1, 1],
-     [1, 0]
-    ].
-
-    For 7 clips, the automatic arrangement becomes:
-    [
-     [1, 1, 1],
-     [1, 1, 1],
-     [1, 0, 0]
-    ].
-
-    For custom arrangements, such as (for 4 clips):
-    [
-     [0, 1, 0, 1],
-     [1],
-     [0, 1]
-    ],
-    the rows will be auto-padded with `0` to be the same length.
+    where `n` is equivalent to ``math.ceil(math.sqrt(len(clips))``\ .
+
+    | For example, for 3 clips, the automatic arrangement becomes:
```

Comment:
In Sphinx, you can start a code block by ending the previous paragprah with `::` and indenting the following text by … probaby 4 spaces?

Reply:
I'll need louis to test building the docs but yea, this seems like a better idea.

Reply:
`python setup.py build_sphinx` then open `build/sphinx/html/inde.html` in a browser